src/vivarium_gates_mncnh/components/lbwsg.py

```diff
@@ -150,6 +150,24 @@ def setup(self, builder: Builder) -> None:
             f"{self.name}.correlated_propensity"
         )
 
+        required_columns = get_lookup_columns(
```

Comment:
These lines are copied and pasted from how the exposure pipelines are defined in LBWSGRisk, with the only difference being the pipeline names.